src/ensembl/src/content/app/custom-download/containers/content/CustomDownloadContent.tsx

```diff
@@ -1,40 +1,60 @@
 import React from 'react';
 import { connect } from 'react-redux';
-import styles from './CustomDownloadContent.scss';
+import { RootState } from 'src/store';
 import {
-  getSelectedTab,
   getShowPreviewResult,
-  getPreviewResult
+  getShowExampleData
 } from '../../state/customDownloadSelectors';
-
 import AttributesAccordion from './attributes-accordion/AttributesAccordion';
 import FiltersAccordion from './filter-accordion/FiltersAccordion';
-import TabButtons from './tab-buttons/TabButtons';
-import ResultHolder from './result-holder/ResultHolder';
-import { RootState } from 'src/store';
+import ResultLoader from './result-loader/ResultLoader';
+import Overlay from '../../components/overlay/Overlay';
+import Panel from '../../components/panel/Panel';
+import PreviewCard from 'src/content/app/custom-download/containers/content/preview-card/PreviewCard';
 import PreviewDownload from './preview-download/PreviewDownload';
-import JSONValue from 'src/shared/types/JSON';
+import { getLaunchbarExpanded } from 'src/header/headerSelectors';
+import { setShowExampleData } from 'src/content/app/custom-download/state/customDownloadActions';
 
-type Props = {
-  selectedTab: string;
+import styles from './CustomDownloadContent.scss';
+
+type StateProps = {
   showSummary: boolean;
-  preview: JSONValue;
+  showExampleData: boolean;
+  launchBarExpanded: boolean;
 };
 
+type DispatchProps = {
+  setShowExampleData: (showExampleData: boolean) => void;
+};
+
+type Props = StateProps & DispatchProps;
+
 const Content = (props: Props) => {
```

Comment:
Name of the default exported component should be the same as the name of the file.

Reply:
That's done